server/app.js

```diff
@@ -109,98 +114,99 @@ const esClient = new elasticsearch.Client({
       port: process.env.ELASTICSEARCH_PORT
     }
   ]
-});
+})
 
 // Setup logger
-app.use(morgan(':remote-addr - :remote-user [:date[clf]] ":method :url HTTP/:http-version" :status :res[content-length] :response-time ms'));
+app.use(morgan(':remote-addr - :remote-user [:date[clf]] ":method :url HTTP/:http-version" :status :res[content-length] :response-time ms'))
 
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true }))
 
 // redirect http to https
-if (process.env.NODE_ENV === "production") {
-  app.enable('trust proxy');
-  app.use(function(req, res, next) {
+if (process.env.NODE_ENV === 'production') {
+  app.enable('trust proxy')
+  app.use(function (req, res, next) {
     if (req.headers['x-forwarded-proto'] && req.headers['x-forwarded-proto'].toLowerCase() === 'http') {
-      return res.redirect('https://' + req.headers.host + req.url);
+      return res.redirect('https://' + req.headers.host + req.url)
     }
-    return next();
-  });
+    return next()
+  })
 }
 
 app.get('/health', (req, res) => {
-  res.json({ success: true });
-});
+  res.json({ success: true })
+})
 
 // if in production, and .htpasswd file exists, set up authentication
 if (process.env.NODE_ENV === 'production' && fs.existsSync(htpasswdFilePath)) {
   const basic = auth.basic({
     file: htpasswdFilePath
-  });
-  app.use(auth.connect(basic));
+  })
+  app.use(auth.connect(basic))
 }
 
 if (process.env.NODE_ENV === 'production' && process.env.PRERENDER_TOKEN) {
-  prerendercloud.set('prerenderToken', process.env.PRERENDER_TOKEN);
-  app.use(prerendercloud);
+  prerendercloud.set('prerenderToken', process.env.PRERENDER_TOKEN)
+  app.use(prerendercloud)
 }
 
 // Serve static assets
 // let index fall through to the wild card route
-app.use(express.static(path.resolve(__dirname, '..', 'build'), { index: false }));
-
+app.use(express.static(path.resolve(__dirname, '..', 'build'), { index: false }))
 
 app.get('/api/objects/:object_id', (req, res) => {
   const options = {
     index: process.env.ELASTICSEARCH_INDEX,
     type: 'object',
     id: req.params.object_id
   }
-  esClient.get(options, function(error, esRes) {
+  esClient.get(options, function (error, esRes) {
     if (error) {
       console.error(`[error] esClient: ${error.message}`)
       res.json(error)
     } else {
       res.json(esRes._source)
     }
   })
-});
+})
 
 app.get('/api/search', (req, res) => {
   esClient.search({
     index: process.env.ELASTICSEARCH_INDEX,
-    body: req.query.body,
-  }, function(error, esRes) {
+    body: req.query.body
+  }, function (error, esRes) {
     if (error) {
-      res.json(error);
+      res.json(error)
     } else {
-      res.json(esRes);
+      res.json(esRes)
     }
-  });
-});
+  })
+})
+
+function getObjectDescriptors (objectID) {
+  const descriptorFields = MORE_LIKE_THIS_FIELDS.concat('people')
```

Comment:
hardcoded people, this distance calculation should maybe be in its own module in the future, and refactored...